templ_enum.cif

```diff
@@ -2158,7 +2158,7 @@ save_wyckoff_letter
          x
          y
          z
-         \a
+         α
```

Comment:
This actually changes the enumeration value from `\a` to `α` thus breaking backwards compatibility with CIF1. We could probably include `α` as an alias of `\a`, but `\a` must be retained.